publication-commons/src/test/resources/features/AccessedToPublishedMaterial.feature

```diff
@@ -0,0 +1,64 @@
+Feature:
+  1. DynamoDBPublicationService allows a user to read any document that is marked as published and
+  therefore it is publicly available.
+
+  2. DynamoDBPublicationService does not allow any user to modify or delete any document that is marked
```

Comment:
While I would think this is a natural assumption, any owner, curator, editor or Admin can modify a published publication associated with their institution. This is because there needs to be an opportunity of correcting mistakes.

Consider also that deletion of resources with status PUBLISHED is considered to be different to other statuses, where a soft/logical deletion takes place in the former case and a physical deletion in the latter. This is not implemented as far as I can see.

Note that this means that the major distinction between the roles is one of access scope to documents in the workflow, rather than one of CRUD privileges. I will try to codify this statement in a document.